### PR TITLE
fix: pagination inside incoming founds table

### PR DESCRIPTION
**Pull request by adam-strzelec**
## Description

Fix pagination inside `Incoming funds` table

## Testing

-Send multiple payments to a colony of the same token, 16 or more.
-Navigate to the Incoming funds page.
-Expand the token to see the individual incoming funds.
-Pagination should work now properly

https://github.com/user-attachments/assets/4445b75c-6799-4965-aeb7-4c7f8630b23f

## Diffs

**Changes** 🏗

Add pagination props for table

Resolves https://github.com/JoinColony/colonyCDapp/issues/3282

### CODE REVIEW

**Review by iamsamgibbs — APPROVED** [review]
Nice job, this is all working as expected:

https://github.com/user-attachments/assets/a4732d99-8189-4085-8a7f-bf997dd677f2

Claiming all still works properly too.

**Comment by rdig** [comment]
@adam-strzelec please wait for 3 accepted reviews before merging your PRs

**Comment by adam-strzelec** [comment]
@rdig Ok, I will know for the future

**Pull request merged**
